draft-ietf-oauth-jwk-thumbprint-uri.xml

```diff
@@ -129,6 +129,32 @@
         The security considerations of <xref target="RFC7638"/>
 	also apply when using this specification.
       </t>
+
+      <section anchor="MultiplePublicKeysPerPrivateKey" title="Multiple Public Keys per Private Key">
+	<t>
+	  There are cryptographic algorithms for which multiple public keys correspond to the same private key.
+	  This is described in the security considerations of <xref target="RFC7748"/> as follows:
+	</t>
+	<t>
+	  <list style="empty">
+	    <t>
+	      Designers using these curves should be aware that for each public
```

Comment:
I think this should say "per each private key"?

Reply:
I believe the authors are saying that other equivalent public keys can be computed from a given public key.

Besides, this is a quotation from another RFC so we can't change it.

Reply:
right, ok